<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobTypeActionsSelectorDialog</class>
 <widget class="QDialog" name="JobTypeActionsSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="CInDocTableView" name="tblActionTypes"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFindByCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string> Поиск по коду или наименованию</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtFindByCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkFindFilter">
     <property name="text">
      <string>Фильтр</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="edtFindFilter">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkOnlyNotExists">
     <property name="text">
      <string>Отсутствующие</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CApplyResetDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CApplyResetDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>library/DialogButtonBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblActionTypes</tabstop>
  <tabstop>edtFindByCode</tabstop>
  <tabstop>chkFindFilter</tabstop>
  <tabstop>edtFindFilter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>JobTypeActionsSelectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>JobTypeActionsSelectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFindFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFindFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFindFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFindByCode</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFindFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFindFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFindFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFindByCode</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
